client/src/Components/PlaceDesigner/Widgets/ImageWidget/ImageWidgetEditPanel.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react';
+import { intlShape, injectIntl } from 'react-intl';
+import {
+    Button,
+    TextField,
+} from '@material-ui/core';
+import { withStyles, WithStyles, createStyles } from '@material-ui/core/styles';
+import {
+    PositionEditor,
+    Category
+} from '../../../Common';
+import ImageWidgetShape from "./ImageWidgetShape";
+import { ImageWidget } from "../../../../../../backend/src/models/interfaces";
+import { UploadImageDialog } from '../../../Dialogs';
+import EditImage from '../../../EditImage';
+
+const styles = createStyles(theme => ({
```

Comment:
use this `const styles: StyleRulesCallback = (theme: Theme) => ({` instead of `createStyles`. and after that remove `const classes: any = this.props.classes;`